DevEdu.API/DevEdu.API/Models/InputModels/PaymentInputModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+
+namespace DevEdu.API.Models.InputModels
+{
+    public class PaymentInputModel
+    {
+        [Required]
+
+        public int Summ { get; set; }
+        public int User { get; set; }
+        public int IsPaid { get; set; }
+

```

Comment:
and what about Date?
does the insert procedure work without such value taken from inputModel?

Reply:
still actual